src/mscorlib/src/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -57,7 +68,67 @@ protected AssemblyLoadContext()
         internal AssemblyLoadContext(bool fDummy)
         {
         }
-        
+#else // !FEATURE_COLLECTIBLE_ALC
+        [System.Security.SecuritySafeCritical]
+        protected AssemblyLoadContext()
+            : this(true) // TODO: Testing only, should default to false!
+        {
+        }
+
+        protected AssemblyLoadContext(bool isCollectible)
```

Comment:
You'll need to update src/mscorlib/model.xml with any new public/protected APIs, otherwise I believe the new APIs will be stripped out prior to publishing. I'm not sure how you would do that though, since the new APIs are enabled conditionally. @weshaggard, do you have a suggestion for this?